.github/workflows/rust.yml

```diff
@@ -23,6 +23,8 @@ env:
   CARGO_INCREMENTAL: 0
   CARGO_NET_RETRY: 10
   RUST_BACKTRACE: short
+  # We allow redundant explicit links because `cargo rdme` doesn't know how to resolve implicit intra-crate links.
+  RUSTDOCFLAGS: -A rustdoc::redundant_explicit_links -D warnings
   RUSTFLAGS: "-D warnings"
```

Comment:
this one has quotes

Reply:
I've removed them for consistency.  YAML doesn't require quotes here, though there's also perhaps an argument that _all_ our strings should have quotes (to avoid [the ‘Norway problem’](https://hitchdev.com/strictyaml/why/implicit-typing-removed/)).